app/assets/stylesheets/admin/views/_document-collection-group-memberships-index.scss

```diff
@@ -0,0 +1,9 @@
+.app-view-document-collection-group-memberships-index__table {
+  .govuk-table__cell {
+    word-break: break-word;
+
+    &:last-child {
+      width: 7em;
+    }
```

Comment:
I think I would try to avoid specifying explicit widths for textual elements because of variations in how devices might render the text, particularly if it's magnified or some-such. Personally I don't have a problem with the links wrapping onto new lines and I think it happens elsewhere but if it is to be avoided I think applying something like `white-space: nowrap;` to the cell would achieve this.

Reply:
the white-space nowrap is a much nicer touch - thank you! I will update